sycl/include/sycl/ext/intel/experimental/esimd/detail/memory_intrin.hpp

```diff
@@ -260,7 +657,8 @@ __ESIMD_INTRIN void __esimd_raw_send_nbarrier_signal(
 /// @tparam DS is the data size.
 /// @tparam VS is the number of elements to load per address.
 /// @tparam Transposed indicates if the data is transposed during the transfer.
-/// @tparam N is the number of channels (platform dependent).
+/// @tparam N is the SIMD size of operation (the number of addresses to access,
+/// platform dependent).
```

Comment:
I don't quite understand what "platform-dependent" means for `N`, as this is what user passes in the code. Restrictions maybe platform-dependent, but that's normal situation for many other ESIMD API parameters, so I suggest to remove it here and on all other similar places

Reply:
Removed 'platform dependent' comment.